src/cli.ts

```diff
@@ -60,5 +60,14 @@ export async function main(options: Options) {
     sdk: options.sdk ?? true,
   })
   await saveFile(options.out, typeString)
+
+  // Generate and save constraints if requested
+  if (options.constraints) {
+    const constraintsString = generateConstraints(schema)
+    const constraintsFile =
+      options.constraintsOut ?? options.out.replace(".ts", ".constraints.ts")
```

Comment:
set a default value "pocketbase-metadata.ts" in index.ts (same way `--out` options is handled).